conf/mesher.yaml

```diff
@@ -21,3 +21,7 @@ admin: #admin API
 #    match:
 #      status: 200
 #      body: ok
+#appPerfManage:
+#  apmName: skywalking
+#  enable: true
+#  serverUri: 192.168.88.64:11800
```

Comment:
apm:
  tracing:
    serverUri: 192.168.88.64:11800 
    enable: true

Reply:
Edit config struct of apm.